### ISSUE TITLE
Cannot set a breakpoint inside let binding block

### ISSUE THREAD

**Issue by vasily-kirichenko** [Microsoft/visualfsharp#2238]
1. Click anywhere within this let binding https://github.com/fsprojects/VisualFSharpPowerTools/blob/master/src/FSharp.Editing/ProjectSystem/LanguageService.fs#L364-L444
2. The whole `let ...` is selected:

![image](https://cloud.githubusercontent.com/assets/873919/21893545/7282ccd6-d8ec-11e6-8aa5-69a0af0d7592.png)

I've tried to make a minimal repro, but failed. On this snippet it's not reproduced:

```fsharp
module M =
    let _ =
        async { 
            let! _ = async.Return ()
            let x =
                [1..10]
                |> List.map (fun x -> 
                    let _ = 
                        match None with
                        | None ->
                            ()
                        | Some _ -> ()
                    ()
                   )
                |> ignore

                let y = 1
                let z = 2
                y + z
            return x
        }
```

**Comment by forki** [comment]
I can reproduce.

**Comment by forki** [comment]
smaller repro;

    let y = [||]
    let x =
            y
            |> Array.map (fun y ->
                    [1..10]
                    |> List.map (fun x -> 
                        let _ = 
                            match None with
                            | None ->
                                ()
                            | Some _ -> ()
                        ()))

**Comment by forki** [comment]
ok found that bug. will send fix in #2237

**Comment by forki** [comment]
done

![break](https://cloud.githubusercontent.com/assets/57396/21898353/e99ab86a-d8ec-11e6-8541-ef34edf17b13.gif)
